modules/channel-web/src/backend/api.ts

```diff
@@ -36,6 +36,8 @@ const SUPPORTED_MESSAGES = [
   'voice'
 ]
 
+const WEBCHAT_CUSTOM_ID_KEY = 'webchatCustomId'
```

Comment:
The reason why I choose `webchatCustomId` instead of `webchatUserId`, is because the id is stored in the user's attributes. So I thought it was obvious that the id is a user ID.

Reply:
And hopefully, the migration of the channel-web to messaging will solve all this confusion!